homeassistant/components/zwave/__init__.py

```diff
@@ -345,6 +346,12 @@ def network_complete():
                      "have been queried")
         hass.bus.fire(const.EVENT_NETWORK_COMPLETE)
 
+    def network_complete_some_dead():
+        """Handle the querying of all nodes on network."""
+        _LOGGER.info("Z-Wave network is complete. All nodes on the network "
+                     "have been queried, but some node ar mark dead")
```

Comment:
.., but some nodes are marked as dead.

Reply:
My initial thought was also like your suggestion, but the python openZwave repo used this spelling, there for I used it: https://github.com/OpenZWave/python-openzwave/blob/d1f645a37f3d95651b494ddb8f2af311e108c84c/src-api/openzwave/network.py#L1310

I will change that rule and in the documentation.